web/src/components/nomination/LocationInput.vue

```diff
@@ -42,57 +45,27 @@ export default {
   data() {
     return {
       locationFields: ['city', 'prefecture'],
-      prefectures: [
-        { en: 'Aichi', ja: '愛知県' },
-        { en: 'Akita', ja: '秋田県' },
-        { en: 'Aomori', ja: '青森県' },
-        { en: 'Chiba', ja: '千葉県' },
-        { en: 'Ehime', ja: '愛媛県' },
-        { en: 'Fukui', ja: '福井県' },
-        { en: 'Fukuoka', ja: '福岡県' },
-        { en: 'Fukushima', ja: '福島県' },
-        { en: 'Gifu', ja: '岐阜県' },
-        { en: 'Gunma', ja: '群馬県' },
-        { en: 'Hiroshima', ja: '広島県' },
-        { en: 'Hokkaido', ja: '北海道' },
-        { en: 'Hyōgo', ja: '兵庫県' },
-        { en: 'Ibaraki', ja: '茨城県' },
-        { en: 'Ishikawa', ja: '石川県' },
-        { en: 'Iwate', ja: '岩手県' },
-        { en: 'Kagawa', ja: '香川県' },
-        { en: 'Kagoshima', ja: '鹿児島県' },
-        { en: 'Kanagawa', ja: '神奈川県' },
-        { en: 'Kōchi', ja: '高知県' },
-        { en: 'Kumamoto', ja: '熊本県' },
-        { en: 'Kyoto', ja: '京都府' },
-        { en: 'Mie', ja: '三重県' },
-        { en: 'Miyagi', ja: '宮城県' },
-        { en: 'Miyazaki', ja: '宮崎県' },
-        { en: 'Nagano', ja: '長野県' },
-        { en: 'Nagasaki', ja: '長崎県' },
-        { en: 'Nara', ja: '奈良県' },
-        { en: 'Niigata', ja: '新潟県' },
-        { en: 'Ōita', ja: '大分県' },
-        { en: 'Okayama', ja: '岡山県' },
-        { en: 'Okinawa', ja: '沖縄県' },
-        { en: 'Osaka', ja: '大阪府' },
-        { en: 'Saga', ja: '佐賀県' },
-        { en: 'Saitama', ja: '埼玉県' },
-        { en: 'Shiga', ja: '滋賀県' },
-        { en: 'Shimane', ja: '島根県' },
-        { en: 'Shizuoka', ja: '静岡県' },
-        { en: 'Tochigi', ja: '栃木県' },
-        { en: 'Tokushima', ja: '徳島県' },
-        { en: 'Tokyo', ja: '東京都' },
-        { en: 'Tottori', ja: '鳥取県' },
-        { en: 'Toyama', ja: '富山県' },
-        { en: 'Wakayama', ja: '和歌山県' },
-        { en: 'Yamagata', ja: '山形県' },
-        { en: 'Yamaguchi', ja: '山口県' },
-        { en: 'Yamanashi', ja: '山梨県' }],
+      prefectures: [],
+      error: null,
     };
   },
+  mounted() {
+    this.getPrefectures();
+  },
   methods: {
+    getPrefectures() {
```

Comment:
Can we DRY this up? One idea would be to fetch the prefectures and store them in the root vue instance. Another would be to make a helper javascript file.

Reply:
Yeah, I thought about fetching in the index page, but since we don't have global state management, we would have to pass this list of prefectures around as props all the way down to the `LocationInput` and `FindSpeaker` components, so I thought it was not very good either.

I guess we could move it to a helper, but I don't see how abstracting it would dry much, since we still need the asynchronous error handling and data binding repeated in each page, right?

Reply:
Ahh, you mean inject a global variable in the Vue app, similar to `$router` but something like `$locations`? Let me try that

Reply:
Yes, you can access global things with `this.$locations` after adding `locations` to the root instance in main.js